src/resources/business-config.ts

```diff
@@ -73,6 +73,16 @@ export const ConfigJson = [
           description: 'update any changes to a company\'s registered or records office addresses as they occur.'
         }
       ]
+    },
+    dissolutionConfirmation: {
+      entityTitle: 'Company',
+      subTitle: 'company',
+      act: 'Business Corporations',
+      modalTitle: 'Voluntary Dissolution',
+      confirmButtonText: 'Continue with Voluntary Dissolution'
+    },
+    TodoList: {
```

Comment:
Now or in a future commit (no rush), please change this to [camelCase](https://en.wikipedia.org/wiki/Camel_case), as is our code style for variables.

Reply:
PS I'm OK with this entire object (ConfigJson) being PascalCase because it's a "const".